src/seaport.ts

```diff
@@ -2543,8 +2570,12 @@ export class OpenSeaPort {
         useTxnOriginStaticCall: false,
       });
 
+    const exchange = await this._wyvernExchangeForOrderCreationPromise;
+
     return {
-      exchange: WyvernProtocol.getExchangeContractAddress(this._networkName),
+      exchange:
+        exchange ||
+        WyvernProtocol.getExchangeContractAddress(this._networkName),
```

Comment:
this looks like it's going to default to 2.3 if our API doesn't reply with any exchange address. can we have it default to the address passed in wyvernConfig? (could default to this if that isn't set either)

Reply:
sg